server/config.ts

```diff
@@ -89,13 +89,21 @@ export default {
       agent: new AgentConfig(Number(get('TOKEN_VERIFICATION_API_TIMEOUT_RESPONSE', 5000))),
       enabled: get('TOKEN_VERIFICATION_ENABLED', 'false') === 'true',
     },
+    csipApi: {
+      url: get('CSIP_API_URL', 'http://localhost:8082', requiredInProduction),
+      timeout: {
+        response: Number(get('CSIP_API_TIMEOUT_RESPONSE', 20000)),
+        deadline: Number(get('CSIP_API_TIMEOUT_DEADLINE', 20000)),
+      },
+      agent: new AgentConfig(Number(get('CSIP_API_TIMEOUT_RESPONSE', 20000))),
+    },
     prisonerSearchApi: {
       url: get('PRISONER_SEARCH_API_URL', 'http://localhost:8082', requiredInProduction),
       timeout: {
         response: Number(get('PRISONER_SEARCH_API_TIMEOUT_SECONDS', 20000)),
-        deadline: Number(get('PRISONER_SEARCH_API_TIMEOUT_SECONDS', 20000)),
```

Comment:
(Minor) all the others use `_RESPONSE` not `_SECONDS` so `PRISONER_SEARCH_API_TIMEOUT_RESPONSE` especially as the value isn't seconds